lib/qless/middleware/retry_exceptions.rb

```diff
@@ -48,19 +48,18 @@ def on_retry_callback
         @on_retry_callback ||= DEFAULT_ON_RETRY_CALLBACK
       end
 
-      def exponential(base, options = {})
+      # If`factor` is omitted it is set to `delay_s` to reproduce legacy
+      # behavior.
+      def exponential(delay_s, options={})
```

Comment:
I run into the issue on many times. `s` is a (SI standard abbreviation)[http://en.wikipedia.org/wiki/SI_base_unit] (ditto ISO). It is much the same way as `lb`, `ft` etc. OTOH, secs is non-standard.

So, for me choice is between `s` and `seconds`. I do not have firm opinion. Given my _personal_ background, `s` is natural. OTOH, I do not write code for myself. In any case, I would like to see consistent approach across Moz code base (I might need to fix it in a few cases it slipped in). How do we want to balance clarity vs. verbosity?